src/components/ContentView.vue

```diff
@@ -1,15 +1,22 @@
 <template>
-    <div class="bg-white w-full p-3 rounded box-border">
-        <slot></slot>    
-    </div>    
+  <div 
+    class="white w-full p-3 rounded box-border"
+    :style="classes"
+  >
+    <slot></slot>    
+  </div>    
 </template>
 
 <script>
 export default {
-    name: 'contentView',
+  name: 'ContentView',
+  computed: {
+    classes() {
```

Comment:
can you help me understand what is classes() for? I'm assuming it has something to do with vuetify

Reply:
It's not vuetify specific, it's just a way of dynamically applying styling. You can bind :class to either an object or an array of class names.

Reply:
What?? That's incredible. I've been looking for something like this for ages. You're a wizard